### PR TITLE
[SHEL-1310] Update No Seating Chart Copy and Graphic

### PR DESCRIPTION
**Pull request by bsl782**
Jira: [SHEL-1310](https://ticketevolution.atlassian.net/browse/SHEL-1310)

## Why / What

This PR adds the missing seatmap icon and centers the icon and text.

## Testing

I tested this code manually.

## Reviewer Checklist (To be edited and marked by approver)

After approving the PR, the reviewer should check the boxes below. See the [Reviewer Checklist](https://ticketevolution.atlassian.net/wiki/spaces/EN/pages/3015016552/Reviewer+Checklist) company standard for more info on these bullet points.

- [ ] Do you understand the reasoning behind the need for the changes?
- [ ] Given only new or existing tests*, do you feel confident that all changes work as intended and solve the problem at hand?
- [ ] Are all success and failure paths accounted for in the tests? If not, are the omissions explained/justified?
- [ ] If coverage ignores are present, is justification provided inline (or linked to) or is a link to a ticket to address in the future provided?
- [ ] Are you willing to accept the same responsibility for this code as the author?

*\*If existing tests, requester should point out what applies*

## PR Code Coverage

- Changeset: XX%
- Overall: XX%

Measure with `bin/test_cov` or some other means. You can find html coverage levels in `coverage/index.html` or in `stdout`.

After a successful test run, you can also see calculated and merged coverage, per branch, on [Grafana](https://grafana.tevo.com/d/6r9OX0y7k/code-coverage?orgId=1)

[SHEL-1310]: https://ticketevolution.atlassian.net/browse/SHEL-1310?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/seatmaps-client/src/TicketMap/MissingSeatMapLogo.tsx
Comment (tevo-nikita-ivashkov) [inline packages/seatmaps-client/src/TicketMap/MissingSeatMapLogo.tsx:1]: Let's put it into src/icons folder and let's try to find and actual svg for this. Not critical but would be more visually scalable and 10 times smaller in code as a svg <path>.

File: packages/seatmaps-client/src/TicketMap/index.tsx [hunk 1]
```diff
@@ -561,26 +563,32 @@ export class TicketMap extends Component<Props & DefaultProps, State> {
...
+              display: "flex",
+              justifyContent: "center",
+              alignItems: "center",
+              flexDirection: "column",
+              textAlign: "center",
+              flex: "1 0 30%",
             }}
           >
+            {!this.props.missingSeatMapLogo && <MissingSeatMapLogo />}
+            {this.props.missingSeatMapLogo}
```
Comment (tevo-nikita-ivashkov) [inline packages/seatmaps-client/src/TicketMap/index.tsx:584]:
```suggestion
            {this.props.missingSeatMapLogo || <MissingSeatMapLogo />}
```

File: packages/seatmaps-client/src/TicketMap/MissingSeatMapLogo.tsx
Comment (bsl782) [reply packages/seatmaps-client/src/TicketMap/MissingSeatMapLogo.tsx:1]: Done )

**Review by tevo-nikita-ivashkov — APPROVED** [review]
lgtm

**Pull request merged**